books/RayTracingInOneWeekend.html

```diff
@@ -2072,8 +2073,21 @@
         }
         ...
       private:
-        ...
+        int    image_height;    // Rendered image height
+        point3 center;          // Camera center
+        point3 pixel00_loc;     // Location of pixel 0, 0
+    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ C++ highlight
+        int    sqrt_spp;        // Square Root of random samples for each pixel
+        double spp_scale;       // Scale value for the random samples for each pixel
```

Comment:
Following the naming used in Book 3, this should be `recip_sqrt_spp_sqrd`, but I find that cumbersome. Not completely convinced on this name however.

Reply:
How about simply `sample_scale`? Maybe how this value gets computed doesn't really need to be conveyed by the variable name besides it being a scale that we apply to samples (as the comment explains).

Perhaps the specific logic for computing it could be explained by an inline code comment and/or the book text surrounding the listing?